static/app/components/onboarding/useOnboardingData.tsx

```diff
@@ -0,0 +1,15 @@
+import {useContext} from 'react';
+
+import {OnboardingContext} from 'sentry/components/onboarding/onboardingContext';
+
+/**
+ * Custom hook to access and update the selected SDK in the onboarding process.
+ */
+export function useOnboardingData() {
+  const {selectedSDK, setSelectedSDK} = useContext(OnboardingContext);
+
+  return {
+    selectedSDK,
+    setSelectedSDK,
```

Comment:
naming nit:
Every else we reference the name platform (e.g. platformSelection).
I would follow that naming also here to make it less confusing
```suggestion
    selectedPlatform,
    setSelectedPlatform,
```